README.md

```diff
@@ -26,7 +28,7 @@ yarn add @editorjs/list
 Include module at your application
 
 ```javascript
-import List from '@editorjs/list';
+import EditorjsList from '@editorjs/list';
 ```
 
 Optionally, you can load this tool from CDN [JsDelivr CDN](https://cdn.jsdelivr.net/npm/@editorjs/list@latest)
```

Comment:
Let's fix major version to prevent problems in a future 

```suggestion
Optionally, you can load this tool from CDN [JsDelivr CDN](https://cdn.jsdelivr.net/npm/@editorjs/list@2)
```